collector/src/main/java/com/bitmovin/analytics/enums/VideoStartFailedReason.java

```diff
@@ -0,0 +1,8 @@
+package com.bitmovin.analytics.enums;
+
+public enum VideoStartFailedReason {
+    PAGE_CLOSED,
+    PLAYER_ERROR,
+    TIMEOUT,
+    UNKNOWN
+}
```

Comment:
make this a Kotlin class and try something like this
```
enum class VideoStartFailedReason(val reason: String) {
    PAGE_CLOSE("page_close")
}
```
This way it is much easier to access the string value of the enum `VideoStartFailedReason.PAGE_CLOSE.reason`